src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/OtlpExporterSpecEnvVarKeyDefinitions.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+namespace OpenTelemetry.Exporter;
+
+/// <summary>
+/// Contains spec environment variable key definitions for OpenTelemetry Protocol (OTLP) exporter.
+/// </summary>
+/// <remarks>
+/// Specification: <see href="https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md"/>.
+/// </remarks>
+internal static class OtlpExporterSpecEnvVarKeyDefinitions
```

Comment:
We could probably rename it to just `SpecEnvVarKeyDefinitions` for brevity. This file would be in the OTLPExporter project anyway.

Reply:
I renamed it on https://github.com/open-telemetry/opentelemetry-dotnet/pull/5429 to try and make it shorter 🤣